Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -150,17 +204,26 @@ function showFormProcessing() {
 </form:form>
 
 <hr>
-
+<c:if test="${not empty conceptSearchBean.foundConcepts}">
 <h3>Results</h3>
 
-<c:if test="${not empty conceptSearchBean.foundConcepts}">
+    <sec:authorize access="isAuthenticated()">
+        <div class="row">
+            <div class="col-sm-2">
+                <button id="prepareMergeConcept" class="btn-sm btn-action">Prepare Merge Concepts</button>
+                <button id="mergeConcept" class="btn-sm btn-action" onclick="mergeConcepts();">Merge Concepts</button>
```

Comment:
If the buttons still stick to the below table as they did when i saw it on your computer, make sure to put some padding there.